package.json

```diff
@@ -19,14 +22,18 @@
   "author": "Vandebron <digital@vandebron.nl>",
   "license": "ISC",
   "peerDependencies": {
-    "@typescript-eslint/eslint-plugin": ">= 2",
-    "@typescript-eslint/parser": "^2.1.0",
-    "eslint-config-prettier": "^6.2.0",
+    "@typescript-eslint/eslint-plugin": "^4.0.0",
+    "@typescript-eslint/parser": "^4.0.0",
+    "eslint": "^6.5.1",
+    "eslint-config-prettier": "^6.4.0",
     "eslint-plugin-import": "^2.18.2",
-    "eslint-plugin-prettier": ">= 3.1",
-    "eslint-plugin-react-hooks": "^2.2.0",
+    "eslint-plugin-prettier": "^3.3.0",
     "eslint-plugin-react": "^7.14.3",
-    "eslint": "^6.3.0",
-    "prettier": "1.18.2"
+    "eslint-plugin-react-hooks": "^2.2.0",
+    "prettier": "^2.2.1",
```

Comment:
Prettier normally recommends pinning an exact version. As this is a package it might however be that we could indeed give it a flexible version, but I"m not sure what the best practice is here.

Reply:
Given it was pinned before, let's pin it again